app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/TicketEphemeralKeyProvider.java

```diff
@@ -28,20 +28,18 @@
 
     private @NonNull ProgressListener mProgressListener;
     private Context mContext;
-    private String mCustomerMail;
 
     public TicketEphemeralKeyProvider(@NonNull ProgressListener progressListener,
-                                      Context context, String customerMail) {
+                                      Context context) {
         mProgressListener = progressListener;
         mContext = context;
-        mCustomerMail = customerMail;
     }
 
     @Override
     public void createEphemeralKey(@NonNull @Size(min = 4) String apiVersion,
                                    @NonNull final EphemeralKeyUpdateListener keyUpdateListener) {
         try {
-            TUMCabeClient.getInstance(mContext).retrieveEphemeralKey(mContext, "2017-06-05", mCustomerMail, new Callback<HashMap<String, Object>>() {
+            TUMCabeClient.getInstance(mContext).retrieveEphemeralKey(mContext, "2017-06-05", new Callback<HashMap<String, Object>>() {
```

Comment:
This is the api version. I because of whatever reason had put the static api version string in there for testing, but we can actually use the apiVersion parameter that the function gets passed.
Changed that.